src/reference-manual/types.Rmd

```diff
@@ -1386,6 +1386,43 @@ As with other assignments, promotions will happen if necessary (of
 `int` to `real` and of `real` to `complex`, along with the
 corresponding container type promotions).
 
+### Unpacking assignment of tuples {-}
+
+For convience of using values stored in tuples, Stan supports
+"unpacking" (or "destructuring") of tuples in an assignment statement.
+
+Given a tuple `T` of type `tuple(T1, ..., Tn)` and a sequence of
+variables of types `v1`, ..., `vn`, where each `vi` has a type which is
+[assignable](#assignment-typing) from type `Ti`, individual elements of
+the tuple may be assigned to the corresponding variables in the sequence
+by the statement
```

Comment:
Would it help to say these aren't general lvalues but must be indexes?  That is, we can't have indexed expressions on the LHS can we?

Would it also help to say that the parens are required, unlike in Python?